org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/variables/ThreadObjectReference.java

```diff
@@ -1,69 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2017 Microsoft Corporation and others.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *     Microsoft Corporation - initial API and implementation
- *******************************************************************************/
-
-package org.eclipse.jdt.ls.debug.adapter.variables;
-
-import java.util.Objects;
-
-import com.sun.jdi.ObjectReference;
-import com.sun.jdi.ThreadReference;
-
-public class ThreadObjectReference {
```

Comment:
This class is changed which changes the implementation of related request handling logic. @andxu please review the new implementation.